fe/fe-core/src/main/java/com/starrocks/catalog/HiveView.java

```diff
@@ -113,4 +119,68 @@ public String getInlineViewDef() {
     public String getCatalogName() {
         return catalogName;
     }
+
+    @Override
+    public String getUUID() {
+        return String.join(".", catalogName, dbName, name, Long.toString(createTime));
```

Comment:
This will cause performance issue on query with information_schema, @letian-jiang  is working on this.

Reply:
better to `return String.join(".", catalogName, dbName, name);`